POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Services/NotificationService.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Windows.UI.Notifications;
+using Microsoft.Toolkit.Uwp.Notifications; // Notifications library
+using Microsoft.QueryStringDotNET; // QueryString.NET
+
+namespace PowerToys_Settings_Sandbox.Services
+{
+    internal class NotificationService
```

Comment:
Usually it's a good idea to add descriptions to your classes/methods/class variables to create clarity, share knowledge with other people who maintain the code, and as reminder for forgetful yourself-in-the-future :)
I personally would not enforce this right now, but it's a good practice and I would expect it for your upcoming main project.